Autopilot/AttitudeManager/fetchSensorMeasurementsMode.cpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * Fetch Sensor Measurements Mode: gets raw sensor data from the sensor drivers
+ * Author: Dhruv Rawat
+ * Last edited: January, 2021
+ */ 
+
+#include "fetchSensorMeasurementsMode.hpp"
+#include <math.h>
+
+#include <iostream>
+
+SensorError_t SensorMeasurements_GetResult(IMU *imusns, airspeed *airspeedsns, IMU_Data_t *imudata, Airspeed_Data_t *airspeeddata) {
+
+    SensorError_t error;
+    error.errorCode = 0;
+
+    // Structures from sensor driver files
+    IMUData_t tempIMUdata;
```

Comment:
why create these temp variables rather than passing in imudata and airspeeddata ? Seems unnecessary

Reply:
I remember why now.

I wanted to basically remove the `#include "IMU.hpp"` and `"#include "airspeed.hpp"` from as many files as possible. One way I did this was by creating new structures that store the imu and airspeed data.

Now, the sensor drivers' `GetResults()` methods do not accept the new structure I made, so I needed to create temporary IMUData_t and airspeedData_t structures to get the data before initializing the new structure objects I made

Reply:
If you'd like, I could just use the IMUData_t and airspeedData_t objects from the sensor drivers. Only downside is SensorFusion will need `#include "IMU.hpp"` and all